models/fixtures/repository.yml

```diff
@@ -217,7 +222,7 @@
   owner_name: user2
   lower_name: repo15
   name: repo15
-  is_empty: true
+  is_empty: false
```

Comment:
This repo contains commits, so it is not empty. `user30/empty` is the new empty repo.